CMakeLists.txt

```diff
@@ -139,56 +139,29 @@ endif()
 ###############################################################################
 ### FMS
 ###############################################################################
-include(fms_files.cmake)
-add_library(fms ${fms_src_files} ${fms_headers})
-
-list(APPEND _fms_defs_public use_libMPI
-                             use_netCDF
-                             GFS_PHYS
-                             INTERNAL_FILE_NML)
-if(QUAD_PRECISION)
-  list(APPEND _fms_defs_public ENABLE_QUAD_PRECISION)
-endif()
+set(GFS_PHYS ON CACHE BOOL "Enable GFS Physics")
```

Comment:
Are all these changes required in this file? Seems like it involves a lot of changes for FMS. Is that because we are changing to a new version of FMS that has its own cmake build?

Reply:
I pulled in https://github.com/ufs-community/ufs-weather-model/pull/441, so this may be part of that.